### ISSUE TITLE
All angle-brackets are converted to HTML entities in [script] passages

### ISSUE THREAD

**Issue by KimikoMuffin** [Dan-Q/twee2#1]
So, in my .tw2 file, I have the following code as part of a macro:

```Javascript
if (params.length >= 3)
    arr = [ '<abbr title="', params[1], ' (', params[2], ')">', params[0], '</abbr>' ];
```
After compiling, it appears in the HTML file like this:
```Javascript
if (params.length &gt;= 3)
    arr = [ '&lt;abbr title="', params[1], ' (', params[2], ')"&gt;', params[0], '&lt;/abbr&gt;' ];
```
This causes two errors: first, it complains about the `&gt;` in the middle of the conditional there, and second, when I implemented it without the greater-than sign, it literally appears on the page as `<abbr title="bar (baz)">foo</abbr>` instead of `foo` with the hover-text `bar (baz)`

**Comment by fuzzybinary** [comment]
I've fixed this in PR #3.  Just needs to be accepted.

**Comment by Dan-Q** [comment]
Fix approved and merged in. Sorry it took so long!
